src/main/java/com/redhat/devtools/intellij/tektoncd/tkn/TknCli.java

```diff
@@ -69,6 +68,11 @@ public boolean isTektonAware(KubernetesClient client) {
         return client.rootPaths().getPaths().stream().filter(path -> path.endsWith("tekton.dev")).findFirst().isPresent();
     }
 
+    @Override
+    public boolean isTektonTriggersAware(KubernetesClient client) {
+        return client.rootPaths().getPaths().stream().filter(path -> path.endsWith("triggers.tekton.dev")).findFirst().isPresent();
```

Comment:
On only issue if that if you have triggers installed but not pipelines, then the `isTektonAware` will return true but I'm not sure this is relevant

Reply:
I didn't think at this because in the documentation they state you need pipeline before to install triggers so i thought there were some kind of prereq and if you didn't have one you coldn't install the other. I wanted to make a test by uninstalling everything from my cluster but it's like the uninstall process doesn't remove everything or my cluster had some issue. The plugin still see tekton installed. Opening an issue to investigate more about this because if the uninstall doesn't remove the tekton.dev path and we check that to see if tekton is installed we could have problem.